pkg/config/config.go

```diff
@@ -262,9 +262,24 @@ func LoadRemote(url string, expandEnvVars bool, c *Config) error {
 }
 
 // LoadDynamicConfiguration is used to load configuration from a variety of sources using
-// ConfigLoader, this is a templated approach
-func LoadDynamicConfiguration(url string, _ bool, c *Config, fs *flag.FlagSet) error {
-	// will be filled in by future pr
+// dynamic loader, this is a templated approach
+func LoadDynamicConfiguration(url string, expandvar bool, c *Config, fs *flag.FlagSet) error {
+	if expandvar {
```

Comment:
I may be missing some context here about the `loader func` below, but do we need this argument if we're always going to exit? 

Could this failure be handled more explicitly at the caller's site, where we also list other failure modes?